samples/apps/bookinfo/rules/route-rule-reviews-v2-v3.yaml

```diff
@@ -5,11 +5,7 @@ metadata:
 spec:
   destination:
     name: reviews
-  precedence: 1
+  precedence: 2
   route:
   - labels:
-      version: v2
```

Comment:
you can't change this file (it says "-v2-v3") you should change the test to use v3 ?